app/Allocator/Squawk/SquawkAssignmentCategories.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace App\Allocator\Squawk;
+
+class SquawkAssignmentCategories
+{
+    const LOCAL = 'LOCAL';
+    const GENERAL = 'GENERAL';
+
+    const TYPES = [
+        self::LOCAL,
+        self::GENERAL
+    ];
+
+    private function __construct()
```

Comment:
Can probably be removed

Reply:
Made it private so this class cannot be instantiated - it's just there for the constants